frontend/src/components/dashboard/DashboardNode.tsx

```diff
@@ -7,56 +7,74 @@ import DashboardCard from "../utils/DashboardCard";
 import LongCardCell from "../utils/LongCardCell";
 import Term from "../utils/Term";
 
+import { Translate } from "react-localize-redux";
+
 const DashboardNode = () => (
-  <NetworkStatsConsumer>
-    {({ networkStats }) => (
-      <DashboardCard
-        iconPath="/static/images/icon-nodes.svg"
-        title="Nodes"
-        className="node-card"
-      >
-        <Row noGutters>
-          <Col xs="6" md="12">
-            <LongCardCell
-              title={
-                <Term
-                  title={"Nodes online"}
-                  text={"Total number of online nodes. "}
-                  href={
-                    "https://docs.near.org/docs/validator/staking#run-the-node"
+  <Translate>
+    {({ translate }) => (
+      <NetworkStatsConsumer>
+        {({ networkStats }) => (
+          <DashboardCard
+            iconPath="/static/images/icon-nodes.svg"
+            title={translate("model.nodes.title").toString()}
+            className="node-card"
+          >
+            <Row noGutters>
+              <Col xs="6" md="12">
+                <LongCardCell
+                  title={
+                    <Term
+                      title={translate(
+                        "component.dashboard.DashboardNode.nodes_online.title"
+                      ).toString()}
+                      text={
+                        translate(
+                          "component.dashboard.DashboardNode.nodes_online.desc"
+                        ).toString() + " "
```

Comment:
Instead of adding these awkward `+ " "`, let's add this space to the `Term` component implementation once

Reply:
Yes. already fixed.